deployment/CP01CNSP_Deploy.csv

```diff
@@ -1,10 +1,37 @@
-CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes
-SI-1507,,AT-31,,CP01CNSP-SP001-00-SPPENG000,4,1,2015-07-23T17:20:00,2015-10-14T21:23:00,CGMCP-01CNSP-00004,CGVEH-CSPPCP-00005,CGCON-CSPPCP-00005,40.13408,-70.77013,,100,134,
-SI-1507,,AT-31,,CP01CNSP-SP001-02-OPTAAJ000,4,1,2015-07-23T17:20:00,2015-10-14T21:23:00,CGMCP-01CNSP-00004,CGVEH-CSPPCP-00005,CGINS-OPTAAJ-00181,40.13408,-70.77013,,100,134,
-SI-1507,,AT-31,,CP01CNSP-SP001-03-NUTNRJ000,4,1,2015-07-23T17:20:00,2015-10-14T21:23:00,CGMCP-01CNSP-00004,CGVEH-CSPPCP-00005,CGINS-NUTNRJ-00428,40.13408,-70.77013,,100,134,
-SI-1507,,AT-31,,CP01CNSP-SP001-05-VELPTJ000,4,1,2015-07-23T17:20:00,2015-10-14T21:23:00,CGMCP-01CNSP-00004,CGVEH-CSPPCP-00005,CGINS-VELPTJ-11915,40.13408,-70.77013,,100,134,
-SI-1507,,AT-31,,CP01CNSP-SP001-06-DOSTAJ000,4,1,2015-07-23T17:20:00,2015-10-14T21:23:00,CGMCP-01CNSP-00004,CGVEH-CSPPCP-00005,CGINS-DOSTAJ-00351,40.13408,-70.77013,,100,134,
-SI-1507,,AT-31,,CP01CNSP-SP001-07-SPKIRJ000,4,1,2015-07-23T17:20:00,2015-10-14T21:23:00,CGMCP-01CNSP-00004,CGVEH-CSPPCP-00005,CGINS-SPKIRJ-00265,40.13408,-70.77013,,100,134,
-SI-1507,,AT-31,,CP01CNSP-SP001-08-CTDPFJ000,4,1,2015-07-23T17:20:00,2015-10-14T21:23:00,CGMCP-01CNSP-00004,CGVEH-CSPPCP-00005,CGINS-CTDPFJ-00347,40.13408,-70.77013,,100,134,
-SI-1507,,AT-31,,CP01CNSP-SP001-09-FLORTJ000,4,1,2015-07-23T17:20:00,2015-10-14T21:23:00,CGMCP-01CNSP-00004,CGVEH-CSPPCP-00005,CGINS-FLORTJ-01205,40.13408,-70.77013,,100,134,
-SI-1507,,AT-31,,CP01CNSP-SP001-10-PARADJ000,4,1,2015-07-23T17:20:00,2015-10-14T21:23:00,CGMCP-01CNSP-00004,CGVEH-CSPPCP-00005,CGINS-PARADJ-00437,40.13408,-70.77013,,100,134,
+CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes
+AT27,,AT27,,CP01CNSP-SP001-00-SPPENG000,1,1,2015-05-07T19:22:30,2015-05-07T19:30:00,CGMCP-01CNSP-00003,CGVEH-CSPPCP-00005,CGCON-CSPPCP-00005,40.13892,-70.779507,,101,133,Found on surface shortly after recovery; recovered to investigate

```

Comment:
Should the notes say "Found on surface shortly after deployment..." ?

Reply:
Yup, they should